### PR TITLE
PEC Specific annotations keys

### PR DESCRIPTION
**Pull request by cw-dvr8**
The terms in the module are specific to PsychENCODE metadata templates.

@karawoo, @Aryllen Please weigh in on the naming convention I used in the schema IDs and let me know if you think they will work or whether we should discuss further.

@kelshmo, @pitviper6 Would one of you please review the contents of the terms and let me know if I've missed any grants or studies?

### CODE REVIEW

**Comment by pitviper6** [comment]
It's missing the following grants:
U01MH122592 - 3/3 High-resolution mapping of cell type-specific DNA (hydroxy)methylation in the human brain during postnatal development and in psychiatric disease
U01MH122590 - 2/3 High-resolution mapping of cell type-specific DNA (hydroxy)methylation in the human brain during postnatal development and in psychiatric disease
U01MH122591 - 1/3 High-resolution mapping of cell type-specific DNA (hydroxy)methylation in the human brain during postnatal development and in psychiatric disease
R01MH106056 - Higher Order Chromatin and Genetic Risk for Schizophrenia
U01MH122849 - Spatial registration of gene expression in the human brain
U01MH116492 - 1/2 Discovery and validation of neuronal enhancers associated with the development of psychiatric disorders

**Comment by pitviper6** [comment]
@cw-dvr8
Missing studies:
LIBD_szControl
Midbrain Genomics (this is a manuscript, not sure it should be included)

**Review by pitviper6 — CHANGES_REQUESTED** [review]
Found some missing grants/studies that I've included in comments below (sorry about that, I forgot I should've commented here).

File: terms/PsychENCODESpecific/grant.json [hunk 1]
```diff
@@ -0,0 +1,124 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id":"sysbio.PECSpecific-annotations.grant-0.0.1",
+    "description": "Grant number the data is being submitted for",
```
Comment (Aryllen) [inline terms/PsychENCODESpecific/grant.json:4]: I would use the same description for the general purpose grant.
```suggestion
    "description": "Grant number including activity code, institute code, and serial number (e.g. U01MH103392)",
```

File: terms/PsychENCODESpecific/study.json [hunk 2]
```diff
@@ -0,0 +1,125 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id":"sysbio.PECSPecific-annotations.study-0.0.1",
+    "description": "Study",
```
Comment (Aryllen) [inline terms/PsychENCODESpecific/study.json:4]: The general purpose 'study' key has an empty string for the description. I would either copy that and have just `"description": ""` or we should update the general purpose study key to have the same description as this one.

File: terms/PsychENCODESpecific/grant.json [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    "type": "string",
+    "anyOf": [
+        {
+            "const": "R01MH109677",
```
Comment (Aryllen) [inline terms/PsychENCODESpecific/grant.json:12]: These will all need a source.

**Review by Aryllen — CHANGES_REQUESTED** [review]
Comments below. Most are things like the format of the id or general requirements (we were going to ask that all values have descriptions and sources). Since both of these exist as keys outside of the PEC specific module, I would also try to keep them consistent with the existing key descriptions (and/or update the existing key descriptions to be better).

File: terms/PsychENCODESpecific/grant.json (hunk 3 above)
Comment (Aryllen) [reply terms/PsychENCODESpecific/grant.json:12]: If I remember correctly, I believe AD links to the grant as source. If PEC doesn't want to do that, you could just add `"source": ""` to all.

**Comment by cw-dvr8** [comment]
@Aryllen Think I've resolved everything, let me know what you think.

**Review by Aryllen — APPROVED** [review]
I updated the spacing to keep everything lined up. It took me a few tries, but I finally realized there were tabs in the source rows. I'm not sure how you are editing these, but some editors let you set your tab key to insert a number of spaces (often 1 tab = 4 spaces). If your editor of choice has that feature, it might help out in the future.

Overall, looks good to me!

**Review by pitviper6 — APPROVED** [review]

**Pull request merged**